### PR TITLE
RHIROS #181 - Add titles to expanded recommendations

### PR DESCRIPTION
**Pull request by apuntamb**
recomm_title

### CODE REVIEW

File: src/Components/SystemDetail/RecommendationsTable.js [hunk 1]
```diff
@@ -12,10 +12,12 @@ const renderExpandedView = (row) => {
     return (
         <TextContent className='ros-rec-content'>
             <Text component={TextVariants.p}>
+                <Text style={{ fontWeight: 'bold' }}>Detected issues</Text>
                 {row.reason}
             </Text>
```
Comment (kgaikwad) [inline src/Components/SystemDetail/RecommendationsTable.js:17]: Attached screenshot vs mockup [page](https://marvelapp.com/prototype/geig8bh/screen/80186730):
- Blank space between two paragraphs is missing.

File: src/Components/SystemDetail/RecommendationsTable.js [hunk 2]
```diff
@@ -12,10 +12,12 @@ const renderExpandedView = (row) => {
...
             <Text component={TextVariants.p}>
-                Recommendations: {row.resolution}
+                <Text style={{ fontWeight: 'bold' }}>Recommendation</Text>
```
Comment (kgaikwad) [inline src/Components/SystemDetail/RecommendationsTable.js:19]: Instead of adding inline style, could please make use of either strong tag or CSS class?

**Review by kgaikwad — CHANGES_REQUESTED** [review]
@apuntamb, added inline observations.

+ adding @terezanovotna into loop for UX review.

**Comment by apuntamb** [comment]
@kgaikwad Tried using `<strong>` and `<br>`. Is this a good approach to addressing your comment? Thoughts?

**Review by kgaikwad — CHANGES_REQUESTED** [review]
@apuntamb, added few inline changes.

**Review by kgaikwad — APPROVED** [review]
LGTM :+1:

**Comment by terezanovotna** [comment]
Looks great! My only thing was to add a space above Recommendation but I see it is already fixed :)

**Comment by kgaikwad** [comment]
Thank you @terezanovotna & @apuntamb for the change!
Merging this pull-request

**Pull request merged**